### PR TITLE
Rework navigation

### PR DESCRIPTION
**Pull request by itrich**
This PR reworks the navigation as proposed today (2022-06-23) in Mattermost, including the following changes: 
- Remove "w3lc0m3", "Corona Info" nav item
- Move "Impressum" nav item to footer
- Add "Chat" nav item

### CODE REVIEW

**Comment by justinotherguy** [comment]
LGTM!

File: templates/layout.html [hunk 1]
```diff
@@ -47,18 +47,16 @@
   </div>
   <div class="navbar-menu" id="navMenu">
     <div class="navbar-start">
-       <a {% if this._path == '/' %} class="is-active navbar-item"{% else %}class="navbar-item"{% endif %} href="{{ '/'|url }}">w3lc0m3!</a>
```
Comment (paepke) [inline templates/layout.html:50]: You can't come back to home "/" without this link. Suggest keeping this, but with an other name.

**Review by paepke — CHANGES_REQUESTED** [review]

File: templates/layout.html (hunk 1 above)
Comment (itrich) [reply templates/layout.html:50]: You can always click on the icon (which is imho known to most internet users)

Comment (paepke) [reply templates/layout.html:50]: lgtm

Comment (justinotherguy) [reply templates/layout.html:50]: for me the name ("w3lc0me!") is a reason for keeping it; but it's simply a copy of the logo; therefore suggesting to remove it - did we overlook something?

Comment (itrich) [reply templates/layout.html:50]: The leetspeak can also be found on the landing page 😋

Comment (justinotherguy) [reply templates/layout.html:50]: true! :)

**Pull request merged**